tgui/packages/tgui/interfaces/OreRedemption.js

```diff
@@ -192,51 +218,117 @@ const OreLine = (properties, context) => {
-          {ore.value}
-        </Flex.Item>
-      </Flex>
-    </Box>
-  );
+            {ore.name}
+          </Flex.Item>
+          <Flex.Item
+            basis="20%"
+            textAlign="center"
+            color={ore.amount > 0 ? 'good' : 'gray'}
+            bold={ore.amount > 0}
+            align="center"
+          >
+            {ore.amount.toLocaleString('en-US')}
+          </Flex.Item>
+          <Flex.Item
+            basis="15%"
+            textAlign="center"
+            align="center"
+            lineHeight="32px"
+          >
+            <NumberInput
+              value={0}
+              minValue={0}
+              maxValue={Math.min(ore.amount, 50)}
+              stepPixelSize={6}
+              onChange={(_e, value) =>
+                act(ore.value ? 'sheet' : 'alloy', {
+                  'id': ore.id,
+                  'amount': value,
+                })
+              }
+            />
+          </Flex.Item>
+          <Flex.Item basis="20%" textAlign="center" align="center">
+            {ore.value}
+          </Flex.Item>
+        </Flex>
+      </Box>
+    );
+  };
+
+/*
+********* ALLOYS BOX PROPERTIES *********
+*/
+
+const AlloyLine = (properties, context) => {
+  const { act } = useBackend(context);
+  const { ore } = properties;
+  const cleanId = ore.id.replace('$', '');
+  return (
+    <Box className="SheetLine">
+      <Flex width="100%">
+        <Flex.Item basis="7%" align="middle">
+          <Box
+            as="img"
+            src={'sheet-' + (cleanId) + '.png'}
+            verticalAlign="middle"
+            ml="`0rem"
+            />
+            </Flex.Item>
+               <FlexItem
+                  basis="30%"
+                  textAlign="middle"
+                  align="center"
+                >
+                  {ore.name}
+
+                </FlexItem>
+                <Flex.Item
+                  basis="35%"
+                  textAlign="middle"
+                  color={ore.amount > 0 ? 'good' : 'gray'}
+                  align="center"
+                  >
+                    {ore.description}
+                  </Flex.Item>
+          <Flex.Item
+            basis="10%"
+            textAlign="center"
+            color={ore.amount > 0 ? 'good' : 'gray'}
+            bold={ore.amount > 0}
+            align="center"
+          >
+            {ore.amount.toLocaleString('en-US')}
+          </Flex.Item>
+          <Flex.Item
+            basis="20%"
+            textAlign="center"
+            align="center"
+            lineHeight="32px"
+          >
+            <NumberInput
+              value={0}
+              minValue={0}
+              maxValue={Math.min(ore.amount, 50)}
+              stepPixelSize={6}
+              onChange={(_e, value) =>
+                act(ore.value ? 'sheet' : 'alloy', {
+                  'id': ore.id,
+                  'amount': value,
+                })
+              }
+              />
+
+          </Flex.Item>
+  </Flex></Box>
```

Comment:
please have these on two diff line endings and clean up indentation

Reply:
Nice catch. Will fix them tonight and commit them.

Reply:
updated commit 674e60b59dc1ccb66c1f0db9665e4b784fb51378